modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/IntegrationAuthorization.java

```diff
@@ -248,4 +258,68 @@ public void t_getApplicationUsersByRoleAgain() {
     public void t_deleteExperiments() {
         deleteExperiment(experiment);
     }
+
```

Comment:
@abhishekjain88 You will probably need to add these new test cases in IntuitIntegrationAuthorization. Please check.